executor/executor.cc

```diff
@@ -550,7 +579,12 @@ void receive_execute()
 		fail("need_prog: no program");
 	uint64 pos = 0;
 	for (;;) {
+#if GOOS_windows
+		signed int rv = read(kInPipeFd, input_data + pos, sizeof(input_data) - pos);
```

Comment:
What happens without this change?
Duplicating whole line and adding ifdef's is not the best way to fix 1 type.

Reply:
I will use typef for all the types from now on.